4_setup_train_ff_topologies/2025_09_11_uci_full_split/submit.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+#SBATCH --job-name=MakeInterchanges  ## job name
+#SBATCH -p free              ## use free partition
+#SBATCH -t 1-00:00:00
+#SBATCH --nodes=1            ## use 1 node, don't ask for multiple
+#SBATCH --ntasks 1
+#SBATCH --cpus-per-task=32
+#SBATCH --mem-per-cpu=4G     ## ask for 1Gb memory per CPU
+#SBATCH --constraint=fastscratch
+#SBATCH -o stdout.txt
+#SBATCH -e stderr.txt
+
+date
+hn=`hostname`
+echo "Running job on host $hn"
+ncpus = $SLURM_CPUS_ON_NODE
```

Comment:
You don't use this, but fyi you'd need to remove the spaces to assign it to `ncpus`. :-)